packages/pages-shared/__tests__/metadata-generator/parseRedirects.invalid.test.ts

```diff
@@ -273,3 +273,46 @@ test("parseRedirects should reject non-relative URLs for proxying (200) redirect
 		],
 	});
 });
+
+test("parseRedirects should reject '/* /index.html'", () => {
+	const input = `
+/* /index.html 200
+/* /index 200
+/ /index.html
+/ /index
+/* /foo/index.html
```

Comment:
Let's also add some valid rules in here. Testing positive cases with negative cases is always good to ensure it works as expected.
```suggestion
/* /foo/index.html

/* /foo
/* /foo 200
/ /foo
```